DependencyInjection/AzineMailgunWebhooksExtension.php

```diff
@@ -30,6 +34,18 @@ public function load(array $configs, ContainerBuilder $container)
 
         if(array_key_exists(self::PUBLIC_API_KEY, $config))
             $container->setParameter(self::PREFIX."_".self::PUBLIC_API_KEY, $config[self::PUBLIC_API_KEY]);
+        
+        if(array_key_exists(self::TICKET_ID, $config))
```

Comment:
As all new config-keys have default values the $config will always have the corresponding key (and value) defined => these "if"-statements are unnecessary.